test/unit/pipeline/number-nonblank.ts

```diff
@@ -0,0 +1,24 @@
+/*
+	File: test/unit/pipeline/number-nonblank.ts
+	cpuabuse.com
+*/
+
+/**
+ * Tests `numberNonBlank`.
+ */
+
+import { strictEqual } from "assert";
+import { processNumberNonBlank } from "../../../src/pipeline/number-nonblank";
+import { animals as input } from "../../input/pipeline";
+import { animals as expected } from "../../expected/pipeline";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing file extension for "../../expected/pipeline"](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5042286)